editor/components/inserter/style.scss

```diff
@@ -27,6 +27,10 @@
 	&:hover {
 		color: #00aadc;
 	}
+
+	& .insert {
+		margin: 0 auto; /* Safari 10. */
```

Comment:
- Would we want the comment to appear in the generated stylesheet? With `/*` it will, otherwise SASS's `//` could prevent it from being included
- What specific to Safari 10 is this addressing? Might help future maintainers decide whether this can be dropped.